assets/app.js

```diff
@@ -1530,6 +1491,27 @@ new Vue({
         switchRemoveMetatag: function(){
             this.remove_mode_metatag = !this.remove_mode_metatag
         },
+        switchTranslationStatus: function(type, cell){
+            proceed = cell.row['translation_requested'] == 1
```

Comment:
when you defined `translation_requested`, you used a string 1 (`"1"`), but here, you're using integer `1` which isn't very nice. try using triple equal checks (`===`), and try using same type of data :slightly_smiling_face: